src/status_im2/contexts/onboarding/syncing/results/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.contexts.onboarding.syncing.results.view
+  (:require [quo2.core :as quo]
+            [utils.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [utils.re-frame :as rf]
+            [react-native.safe-area :as safe-area]
+            [status-im2.contexts.onboarding.syncing.results.style :as style]
+            [status-im2.contexts.onboarding.common.syncing.render-device :as device]
+            [status-im2.contexts.onboarding.common.background.view :as background]))
+
+(defn page-title
+  []
+  [rn/view {:style {:padding-horizontal 20}}
+   [quo/text
+    {:accessibility-label :notifications-screen-title
+     :weight              :semi-bold
+     :size                :heading-1
+     :style               {:color colors/white}}
+    (i18n/label :t/sync-devices-complete-title)]
+   [quo/text
+    {:accessibility-label :notifications-screen-sub-title
+     :weight              :regular
+     :size                :paragraph-1
+     :style               {:color colors/white}}
+    (i18n/label :t/sync-devices-complete-sub-title)]])
+
+(defn navigation-bar
+  []
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav]])
+
+(defn current-device
+  [installations]
+  [rn/view {:style style/current-device}
+   [device/render-device
+    (merge (first installations)
+           {:this-device? true})]
+   [quo/text
+    {:accessibility-label :notifications-screen-sub-title
+     :weight              :regular
+     :size                :paragraph-1
+     :style               {:color colors/white}}
+    (i18n/label :t/sync-with)]])
+
+(defn render-device-list
```

Comment:
It's better to reserve the word `render` in `render-XYZ` for *things* that should be called (as functions), not components which should be used as Hiccup. In other words, it's better to just name the var `device-list`. The same feedback goes to the `render-device` component in the ns `render-device`, which would be better named just `view`.